datascience-notebook/tests/inspec.yml

```diff
@@ -0,0 +1,9 @@
+---
+name: NGINX Container
+title: NGINX Container
+maintainer: OPG
```

Comment:
These first three lines are a copy/paste slip?

Reply:
Oh yeah totally. Given I wrote them in the first place I only feel lightly embarrassed you caught this 😆